assets/src/styles/blocks/PageHeader.scss

```diff
@@ -55,16 +55,18 @@ $text-column-padding-in: 24px;
     font-size: $font-size-xxl;
     font-weight: 700;
     line-height: initial;
+    margin-inline-start: -$sp-1;
   }
 
-  h1.has-background {
+  .wp-block-heading.has-background {
```

Comment:
I'd prefer to use `classNames` instead of `selectors`. By changing this, we'll avoid to deal with layout issues in cases where the editor change the heading format. I.e. change `h1` to `h2`.